hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheRecordStore.java

```diff
@@ -109,6 +110,7 @@
     public static final UUID SOURCE_NOT_AVAILABLE = UuidUtil.NIL_UUID;
     protected static final int DEFAULT_INITIAL_CAPACITY = 256;
 
+    protected final Timer timer = Timer.getSystemTimer();
```

Comment:
Do we need an instance of the timer? Why not make the methods static? Makes the timer easier to deal with.